src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -69,17 +71,19 @@ export default function AnnotationsControls({
 
     return (
         <div className="bp-AnnotationsControls">
-            <AnnotationsButton
-                data-resin-target="draw"
-                data-testid="bp-AnnotationsControls-drawBtn"
-                isActive={isDrawingActive}
-                isEnabled={showDrawing}
-                mode={AnnotationMode.DRAWING}
-                onClick={handleModeClick}
-                title={__('drawing_comment')}
-            >
-                <IconDrawing24 fill={isDrawingActive ? annotationColor : '#fff'} />
-            </AnnotationsButton>
+            <AnnotationsTargetedTooltip isEnabled={showDrawing}>
```

Comment:
Do we have a preference for which annotation type we want to draw the most attention to? It seems like the choice would be between drawing and regions, the latter of which may be a bit simpler of a user experience for a first-time user. It may be worth checking with the product team on this question.

Reply:
I will confirm with product 👍

Reply:
Thanks, it looks like they prefer the comment region button!